merkledag/merkledag_test.go

```diff
@@ -13,6 +13,8 @@ import (
 	"testing"
 	"time"
 
+	blocks "gx/ipfs/Qmej7nf81hi2x2tvjRBF3mcp74sQyuDH4VMYDGd1YtXjb2/go-block-format"
```

Comment:
This should be grouped with `cid`, `u`, etc. (we group by standard, in-repo, then out-of-repo).